transforms/react-component-rename-props.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable guard-for-in */
+
+// Components whose props shall be renamed
+const componentNamesToRefactor = [
+  // 'TextInput'
```

Comment:
Should this maybe be a CLI argument to the codemod. Is that possible somehow?

Reply:
Maybe like:

```txt
/codemods
   /migrations
      /v10
	      text-input.js
          button.js
```

to then `jscodeshift -- --migration v10/text-input.js

Reply:
Yes, I'd to that as a follow-up, as this PR moves forward and we proceed to refactor all the other components, then list them all those components and props mappings in files that are then imported by this codemod.